LearningMissionLab/Schedule.cs

```diff
@@ -1,10 +1,26 @@
 ﻿using System;
+using System.Collections.Generic;
+
 namespace LearningMissionLab
 {
     public class Schedule
     {
-        public Schedule()
+        ZoomMeeting _zoomMeeting;
+        DateTime _startDate;
+        DateTime _finishDate;
+        List<ScheduleEvent> _scheduleEvent;
+
+        public Schedule(DateTime startDate, DateTime finishDate, ZoomMeeting zoomMeeting, List<ScheduleEvent> scheduleEvent)
         {
+            this._startDate = startDate;
+            this._finishDate = finishDate;
+            this._zoomMeeting = zoomMeeting;
+            this._scheduleEvent = scheduleEvent;
         }
+
+        public ZoomMeeting ZoomMeeting { get => _zoomMeeting; set => _zoomMeeting = value; }
```

Comment:
ZoomMeeting belongs to ScheduleEvent. Each schedule event can use a different zoom link.